chainer/functions/math/erfinv.py

```diff
@@ -0,0 +1,63 @@
+import numpy
+
+import chainer
+from chainer.backends import cuda
+from chainer import function_node
+from chainer import utils
+from chainer.utils import type_check
+
+
+_erfinv_cpu = None
+
+
+class ErfInv(function_node.FunctionNode):
+
+    @property
+    def label(self):
+        return 'erfinv'
+
+    def check_type_forward(self, in_types):
+        type_check.expect(in_types.size() == 1)
+        type_check.expect(in_types[0].dtype.kind == 'f')
+
+    def forward_cpu(self, x):
+        global _erfinv_cpu
+        if _erfinv_cpu is None:
+            try:
```

Comment:
Could you write try-import at the global scope? The exception can be cached and reraised in `forward_cpu`. (see e.g. https://github.com/chainer/chainer/blob/master/chainer/datasets/image_dataset.py#L4-L9)